apps/vue-app/src/app/modules/auth/views/LoginView.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="d-flex justify-content-center align-items-center" style="height: 100vh">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title text-center">Login</h5>
+        <form>
+          <div class="form-group pb-3">
+            <label>Username</label>
+            <input type="text" class="form-control is-invalid" v-model="credentials.username" />
```

Comment:
El input de user siempre se muestra rojo aunque sea válido el contenido, faltó hacer uso de class binding aquí para agregar is-invalid class